routes/mentors.js

```diff
@@ -64,4 +64,27 @@ router.get("/mentors/:id", verifyRoles([ROLE_NAMES.member]),  async (req, res) =
   }
 });
 
+// Update mentor status
+router.patch("/mentors/:id", verifyRoles([ROLE_NAMES.member]), async (req, res) => {
+  const { id } = req.params;
+  const { status } = req.body;
+  if(!isValidIntegerId(id)) {
+    return respondWithError(res, 400, "Invalid user id supplied");
+  }
+  if(!ALLOWED_MENTOR_STATUSES[status]) {
+    return respondWithError(res, 400, "Invalid status supplied");
+  }
+  try {
+    const isOwner = await verifyMentorOwnership(id, req.userId);
+    const mentorId = await mentorService.updateMentorStatus(req.userRoles, id, status, isOwner);
+    res.json(mentorId);
```

Comment:
can we please add simple tests for this route, specifically testing these three cases:
* returning 200
* retuning 403
* returning 404